runtime/compiler/z/codegen/S390J9CallSnippet.cpp

```diff
@@ -1653,3 +1653,67 @@ TR::S390VirtualUnresolvedReadOnlySnippet::getLength(int32_t estimatedSnippetStar
            + sizeof(int32_t); /*RIP offset to post snippet call in mainline*/
    }
 
+uint8_t *
+TR::S390InterfaceCallReadOnlySnippet::emitSnippetBody()
+   {
+   uint8_t *cursor = cg()->getBinaryBufferCursor();
+   getSnippetLabel()->setCodeLocation(cursor);
+   TR::Node *callNode = getNode();
+
+   TR::SymbolReference *interfaceCallReadOnlySymRef = cg()->symRefTab()->findOrCreateRuntimeHelper(TR_S390interfaceCallHelperMultiSlotsReadOnly,
+                                                                                                               false, false, false);
+   
+   TR::GlobalFunctionCallData dataDestination(interfaceCallReadOnlySymRef, callNode, cursor, cg(), self());
+
+   cursor = cg()->getObjFmt()->encodeGlobalFunctionCall(dataDestination);
+
+   uint8_t *helperCallRA = cursor;
+   TR_ASSERT_FATAL(cg()->canUseRelativeLongInstructions(interfaceCallPICSlotDataAddress), "interfaceCallPICData %p is outside relative immediate range", interfaceCallPICSlotDataAddress);
+   *reinterpret_cast<int32_t*>(cursor) = static_cast<int32_t>(interfaceCallPICSlotDataAddress - (intptr_t)helperCallRA);
+   cursor += sizeof(int32_t);
+
+   intptr_t doneLabelAddress = reinterpret_cast<intptr_t>(doneLabel->getCodeLocation());
+   *reinterpret_cast<int32_t*>(cursor) = static_cast<int32_t>(doneLabelAddress - (intptr_t)helperCallRA);
+   cursor += sizeof(int32_t);
+
+   return cursor;
+   }
+
+void
+TR_Debug::print(TR::FILE *pOutFile, TR::S390InterfaceCallReadOnlySnippet *snippet)
+   {
+   uint8_t *bufferPos = snippet->getSnippetLabel()->getCodeLocation();
+   TR::SymbolReference *resolveVirtualDispatchReadOnlySymRef = _cg->getSymRef(TR_S390interfaceCallHelperMultiSlotsReadOnly);
+
+   TR::SymbolReference *methodSymRef = snippet->getNode()->getSymbolReference();
+   // TODO: Need the address of the ccGlobalFunctionCallData   
+   printSnippetLabel(pOutFile, snippet->getSnippetLabel(), bufferPos, "Interface Call Snippet using Multi Slots");
+   printPrefix(pOutFile, NULL, bufferPos, 6);
+   trfprintf(pOutFile, "LGRL \tGPR14, <%p>\t# Load address of the Helper Method, targetAddress = <%p>",
+                    (void *)*(int32_t*)(bufferPos+sizeof(int16_t)),
```

Comment:
`void *` vs `void*` inconsistent. Also need spaces between arithmetic `+` operators.

Reply:
Fixed in https://github.com/eclipse/openj9/compare/ab82c5d254e0315b3c65722371fda75cc1d79a10..3282a01df297676d33999fca192d2613b5db1b09